lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -200,7 +204,7 @@ export function detectTestingLibraryUtils<
           ? importedUtilSpecifier.imported.name
           : undefined;
 
-      if (!isUtilCallback(node.name, originalNodeName)) {
+      if (!isPotentialFunctionCallback(node.name, originalNodeName)) {
```

Comment:
This check is one of the problems I found that I'll addresss separately. Instead of the current check, this should be done using `hasImportMatch` + `referenceNodeIdentifier.name`.

This causes some edge cases using wildcard imports (e.g. `import * as RTL from '@testing-library/react') not to be detected properly.